will/tests/test_plugin.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+import re
+
+import pytest
+from freezegun import freeze_time
+
+from will import settings
+from will.plugin import WillPlugin
+
+# Used to freeze_time
+WILLS_BIRTHDAY = "2013-12-04 01:11:00"
+
+
+@pytest.fixture
+def plugin():
+    return WillPlugin()
+
+
+@pytest.fixture
+def content():
+    return "This is a test content"
+
+
+def test__prepared_content(plugin, content):
+    assert plugin._prepared_content(
+        content, "", "") == re.sub(r'>\s+<', '><', content)
```

Comment:
I think it makes more sense to assert it against a pinned output: "This is a test content"

Reply:
also, I don't think prepared_content is used anywhere

Reply:
yeahh, didn't find it being used anywhere either.

Would it be better to remove prepared_content function from the plugin module?

Or just assert against a pinner output?

I am up for removing unused code

Reply:
removing dead code seems like a good idea 👍 

any input, @rsalmond ?